files/galaxy/dynamic_rules/usegalaxy/tool_destinations.yaml

```diff
@@ -1132,11 +1132,7 @@ spyboat: {cores: 8, mem: 4}
 sshmm: {mem: 16}
 structurefold: {mem: 12}
 rnaspades: {cores: 10, mem: 110}
-spades_biosyntheticspades:
-  cores: 10
-  mem: 400
-  env:
-    GALAXY_MEMORY_GB: 400
```

Comment:
why is this not needed anymore? I guess the old versions still need it

Reply:
I updated the tool wrappers in order to fix it, so I'm not sure if it is necessary at all. Since it affects the whole set of SPAdes tools, it has not so much sense to keep it just for BiosyntheticSPAdes I think. If you consider that it can be useful I can keep it.

Reply:
I'm just thinking about older version of this tool, that still need it. Maybe leave it in for another few month and then we assume that all people are using the new/fixed version?

Reply:
Sure, I'll keep it.